source/json/people.schema.json

```diff
@@ -36,7 +36,25 @@
         }
       },
       "image": {
-        "type": "string"
+        "type": "string",
+        "faker": "internet.avatar"
+      },
+      "internetHealthIssues": {
```

Comment:
can we stick to an `_` convention that Django uses? Would rather not write a bunch of extra code to get it to do camelcasing.